isb_web/config.py

```diff
@@ -1,9 +1,9 @@
 from pydantic import BaseSettings
 
 class Settings(BaseSettings):
-    database_url: str = "UNSET"
+    database_url: str = "postgresql+psycopg2://isb_writer:isamplesinabox@db/isb_1"
     web_root: str = "/"
-    solr_url: str = "UNSET"
+    solr_url: str = "http://solr:8983/solr/isb_core_records"
```

Comment:
I need to figure out why the `.env` file wasn't working here.  I knew I could make it work by hardcoding these for now, and the hostnames match up with the values provided in the containing docker compose file.  Mainly just wanted to make sure that the services could see one another, and they were definitely able to do that.

Reply:
Figured this out.